From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nikolay Amiantov <ab@fmap.me>
Date: Thu, 25 Jul 2019 20:45:55 +0300
Subject: [PATCH] systemd-shutdown: execute scripts in
 /etc/systemd/system-shutdown

This is needed for NixOS to use such scripts as systemd directory is immutable.
---
 src/shutdown/shutdown.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shutdown/shutdown.c b/src/shutdown/shutdown.c
index 5dee1b3a92..c08cf80548 100644
--- a/src/shutdown/shutdown.c
+++ b/src/shutdown/shutdown.c
@@ -339,6 +339,7 @@ static void init_watchdog(void) {
 int main(int argc, char *argv[]) {
         static const char* const dirs[] = {
                 SYSTEM_SHUTDOWN_PATH,
+                "/etc/systemd/system-shutdown",
                 NULL
         };
         _cleanup_free_ char *cgroup = NULL;
